app/models/badges/ewok.rb

```diff
@@ -0,0 +1,19 @@
+class Badge::Ewok
+  BADGE_ID = 5
+  DISPLAY_NAME = 'Ewok'
+  SLUG = 'ewok'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.